### PR TITLE
Fix for missing linux packages and docker-composer check

### PR DESCRIPTION
**Pull request by crajapakshe**

### CODE REVIEW

File: manage [hunk 1]
```diff
@@ -499,7 +499,7 @@ if ! isInstalled ${SED_EXE}; then
 
   cat <<-EOF
   The recommended approach to installing the required package(s) is to use either [Homebrew](https://brew.sh/) (MAC)
-  or [Chocolatey](https://chocolatey.org/) (Windows).  For more information visit https://mikefarah.github.io/yq/
+  or [Chocolatey](https://chocolatey.org/) (Windows) or [GitHub](https://github.com/mikefarah/yq/) (Linux).  For more information visit https://mikefarah.github.io/yq/
```
Comment (WadeBarnes) [inline manage:502]: These should be the instructions for installing `sed`.  The existing reference to `yq` here was a copy and paste mistake.

File: manage [hunk 2]
```diff
@@ -487,7 +487,7 @@ if ! isInstalled ${DOCKER_EXE}; then
   exit 1
 fi
 
-DOCKER_COMPOSE_EXE="${DOCKER_EXE} compose"
+DOCKER_COMPOSE_EXE="${DOCKER_EXE}-compose"
```
Comment (WadeBarnes) [inline manage:490]: This is specifically using the new `docker compose` syntax, rather than the legacy `docker-compose` syntax.  Please revert this change.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
@crajapakshe, Thanks for the contribution.  There are a couple small items to address.

File: manage (hunk 2 above)
Comment (swcurran) [reply manage:490]: But then it is explicitly using the old syntax that doesn’t work everywhere either.  Better still to use the change [here](https://github.com/hyperledger/aries-agent-test-harness/blob/3b8b2eb8f74088bcc6bcbe5457880dbe1ff1ea25/manage#L69) to use the command appropriate for the version of docker in use.

Comment (WadeBarnes) [reply manage:490]: To clarify the PR changed the call from the new `docker compose` syntax to the legacy `docker-compose` syntax.  This project has never used the legacy `docker-compose` syntax.  If I recall, I could be wrong, but I believe there are some features in the scripts that require the new version of docker compose.  There was a reason I started with it, I just can't recall exactly.

File: manage [hunk 3]
```diff
@@ -499,12 +499,14 @@ if ! isInstalled ${SED_EXE}; then
 
   cat <<-EOF
   The recommended approach to installing the required package(s) is to use either [Homebrew](https://brew.sh/) (MAC)
-  or [Chocolatey](https://chocolatey.org/) (Windows).  For more information visit https://mikefarah.github.io/yq/
+  or [Chocolatey](https://chocolatey.org/) (Windows) or [GitHub](https://github.com/mirror/sed) (Linux).  For more information visit https://mikefarah.github.io/yq/
```
Comment (WadeBarnes) [inline manage:502]:
```suggestion
  or [Chocolatey](https://chocolatey.org/) (Windows) or [GitHub](https://github.com/mirror/sed) (Linux).  For more information visit https://www.gnu.org/software/sed/
```

File: manage [hunk 4]
```diff
@@ -499,12 +499,14 @@ if ! isInstalled ${SED_EXE}; then
...
 
   Windows:
     - chocolatey install sed
   MAC:
     - brew install gnu-sed
+  Linux (Ubuntu):
+    - apt install sed
```
Comment (WadeBarnes) [inline manage:509]:
```suggestion
  Debian/Ubuntu:
    - sudo apt-get install sed
```

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
Some small recommendations below.  Could you also squash all the changes into a single commit to clean up the history please?

**Comment by crajapakshe** [comment]
> Some small recommendations below. Could you also squash all the changes into a single commit to clean up the history please?

@WadeBarnes Updated.

File: manage (hunk 3 above)
Comment (WadeBarnes) [reply manage:502]: You still need to change the link reference from `https://mikefarah.github.io/yq/` to `https://mikefarah.github.io/yq/`

File: manage (hunk 4 above)
Comment (WadeBarnes) [reply manage:509]: I'm making this recommendation to keep things consistent with the changes you made in `fetch-validator-status/run.sh`

**Review by WadeBarnes — CHANGES_REQUESTED** [review]

**Comment by WadeBarnes** [comment]
My latest review comments make more sense in context as comments on the preview review comments where I actually made the suggested changes.  It seems GitHub has yanked them out of context in the most recent review.

**Review by WadeBarnes — APPROVED** [review]

**Comment by crajapakshe** [comment]
> My latest review comments make more sense in context as comments on the preview review comments where I actually made the suggested changes. It seems GitHub has yanked them out of context in the most recent review.

Thanks for pointed out and update the code with suggestions.
